### PR TITLE
Issues2

### PR DESCRIPTION
**Pull request by HenintsoaHARINORO**
Pascal 's Triangle

### CODE REVIEW

File: templates/triangle.html.twig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <div class="card" style="background-color: #17a2b8">
+            <form action="triangle2.php" method="post">
+                Nombre:<input type="text" name="number"> <br><br>
+                <input type="submit" value="Calculer"><br><br>
+
+           {% for i in j %}
+                {% for line in output %}
+
+                {% endfor %}
+           {% endfor %}
```
Comment (HenintsoaHARINORO) [inline templates/triangle.html.twig:13]: I do not know how to print it here as two for loops
@Legion112

File: triangle2.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    $c=$_POST["number"];
+    $output=[];
+        for ($j = 0; $j <= $c; $j++) {
+            for ($i = 0; $i <= $c; $i++) {
+                if ($j >= $i) {
+                    $output[]=[
+                        'a'=>' ',
+                        'b'=>combinaison($i,$j),
+                        'c'=>' '
+                    ];
```
Comment (HenintsoaHARINORO) [inline triangle2.php:40]: I do not know if I can do that

**Review by HenintsoaHARINORO — COMMENTED** [review]
Refactoring Pascal's Triangle

**Comment by Legion112** [comment]
If you do
```
git log
```
You will see
```
commit b9e8bab13a0919c4d184019242f9d39d4c1cba06 (HEAD -> issues2, origin/issues2)
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Sat Mar 27 13:27:34 2021 +0300

    refactoring

commit 26c2208574ef902795ed0b887523015f9e92e3e8
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 22:42:31 2021 +0300

    finished

commit bd55827e64b549afdc7bd868bba646e9129b6b6b (origin/issues1, issues1)
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 22:15:46 2021 +0300

    fixes

commit 7825cbfb98b1bab129b22379859fbc174c71a10a
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 22:10:48 2021 +0300

    s

commit c6ca86a556b4ab258ca3fb9381d325f78c352dab
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 21:54:59 2021 +0300

    changes

commit a6d66ffd0178c11d5dd773ce002098f7b615cf08
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 21:45:34 2021 +0300

    changes

commit df39e767bd4cf7f23b323ee298ebe993ccfe3468
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 20:24:24 2021 +0300

    fix something

commit bb42363d63f4c33006362f3a1e229950a2ec92f4
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 19:12:08 2021 +0300

    removed cache

commit 378814905b327d25d6576070c5bf50fbb67a0aeb
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Fri Mar 26 18:47:58 2021 +0300

    getting input

commit dfffde8b663eaa6b32fd240c4dd835d1c1971853
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Thu Mar 25 17:19:14 2021 +0300

    bootstrap external file

commit a6ce05a252183bc45a098c1c03dedeea7faa56c3
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Thu Mar 25 17:15:54 2021 +0300

    removed bootstrap

commit f5c1fc4ed20041509f22f661404b1b5f0055651a
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Thu Mar 25 11:24:23 2021 +0300

    boostrap

commit 1947a3c2b49b0fd99f617b942000fae7f09a1f7b
Author: HenintsoaHARINORO <henintsoaharinororandrianasolo@gmail.com>
Date:   Thu Mar 25 11:21:29 2021 +0300

    using twig

commit 7f4571b302bc5db909f12d923507d5d28f39de48 (main, issues/1, feature-1)
Merge: 593d386 5e22c41
Author: Henintsoa Harinoro <48785203+HenintsoaHARINORO@users.noreply.github.com>
Date:   Tue Mar 16 17:36:13 2021 +0300

    Merge pull request #2 from Legion112/refactoring
    
    Refactoring
```
This is the list of your commits. 
I see you have created this branch from a branch named **issues1** it is wrong.
Each new branch you have to create from **main** branch.
Github has a page that shows the difference between your branch and master.
[Here](https://github.com/HenintsoaHARINORO/ExoPHP/pull/8/commits)

**Comment by Legion112** [comment]
You could fix it by. 
Rewriting the history of your commits.
```
git rebase -i HEAD~13
```
In the open editor mark, all commits that are not related to this branch as `drop` or `d`
```
d 1947a3c using twig
d f5c1fc4 boostrap
d a6ce05a removed bootstrap
d dfffde8 bootstrap external file
d 3788149 getting input
d bb42363 removed cache
d df39e76 fix something
d a6d66ff changes
d c6ca86a changes
d 7825cbf s
d bd55827 fixes
d 26c2208 finished
pick b9e8bab refactoring
```
After push your changes with flag --force
```
git push --force
```
As the result, you will see only a single commit [here](https://github.com/HenintsoaHARINORO/ExoPHP/pull/8/commits)'

**Pull request closed without merging**